api/src/main/java/com/nicknathanjustin/streamercontracts/users/View.java

```diff
@@ -0,0 +1,5 @@
+package com.nicknathanjustin.streamercontracts.users;
+
+public class View {
```

Comment:
What is this used for?

Reply:
I think this shouldnt be needed if we construct our modelObjects at the controller level differently (see above comment)